api/v1alpha1/replicationdestination_types.go

```diff
@@ -166,6 +169,13 @@ type ReplicationDestinationRsyncStatus struct {
 	Port *int32 `json:"port,omitempty"`
 }
 
+// ReplicationDestinationResticSpec defines the field for restic in replicationDestination.
+type ReplicationDestinationResticSpec struct {
```

Comment:
@JohnStrunk  should `ReplicationDestinationResticSpec` have `Retain` ? Both source and dest access the same restic-repo, should restic job doing `restore` do `forget`